packages/components/src/components/hds/form/super-select/multiple/base.hbs

```diff
@@ -59,6 +59,7 @@
     @search={{@search}}
     @searchEnabled={{@searchEnabled}}
     @searchField={{@searchField}}
+    @searchFieldPosition={{@searchFieldPosition}}
```

Comment:
Ultimately, we most likely will want to hard code this to "before-options" but I will do some testing within consumer products to see if we could release with that initially or if a phased release approach is better.

Reply:
I added the results of testing in the ticket: https://hashicorp.atlassian.net/browse/HDS-4254

To me, it seems fairly safe to hard-code the value as "before-options" for release. We could perhaps post in Slack channels to double-check with consumers first though and give them a heads-up about the change.

Reply:
If tests are not failing, and it seems like they aren't, we can safely hard-code it. Indeed, a good idea to give folks a heads-up about the change in our public channel.